packages/data-models/appConfig.ts

```diff
@@ -200,6 +200,14 @@ const ASSET_PACKS = {
   folderName: "asset_packs",
 };
 
+const TASKS = {
+  enabled: true,
+  // The name of the field that tracks the currently highlighted task group
+  highlightedTaskField: "_task_highlighted_group_id",
+  // The top-level list of task groups (e.g. modules)
+  taskGroupsListName: "workshop_tasks",
```

Comment:
nit(question)
I'm also not 100% sure what role the task groupList name plays - is this something that needs to be hardcoded into the app or something that should be passed to a `task_group` component (can it only ever be one fixed value so one task list per app??). If it is a default perhaps just `defaultGroupName` ?

Reply:
Good point, see reply in [this mattermost thread](https://chat.idems.international/all/pl/jps5afo4c7r69qsp8o74o37d5c)

Reply:
Will also resolve here and link to:
https://docs.google.com/document/d/1qVcUri2DAhyiFx6ihtFLeVeJjoS1ba0rdxntRg3AGec/edit?disco=AAABC5ZxhSw